index.js

```diff
@@ -40,7 +49,83 @@ function toggleColor(element) {
   }
 }
 
+const header = document.querySelector("h1#header")
+
+header.addEventListener("click", function() {
+  toggleColor(document.querySelector("h1#header"))
```

Comment:
Looks good! Here, you can also use the `header` variable instead of calling `document.querySelector("h1#header")` again.